common_factories.py

```diff
@@ -201,7 +201,7 @@ def getBuildFactoryPreTest(build_type="RelWithDebInfo", additional_args=""):
                         "additional_args", default=f"{additional_args}"
                     ),
                     create_package=util.Property("create_package", default="package"),
-                    verbose_build=util.Property("verbose_build", default=""),
+                    verbose_build=util.Property("verbose_build", default="VERBOSE=1"),
```

Comment:
with #606 merged I think `--output-sync=target` can be added to this as no bintar builder uses f_quick_build.